format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -0,0 +1,91 @@
+package org.scalafmt.internal
+
+import scala.meta._
+import scala.meta.Token
+import scala.meta.tokens.Token._
+
+object SyntaxTokens {
+  private implicit class Util[A <: Tree](val tree: A) extends AnyVal {
+    def findAfter(f: Token => Boolean, p: A => Tree): Option[Token] = {
+      val end = p(tree).tokens.last
+      tree.tokens.dropWhile(_ != end).drop(1).find(f)
+    }
+  }
+  implicit class TermAssignSyntax(val tree: Term.Assign) extends AnyVal {
```

Comment:
To follow scalameta/scalafix conventions, can we please use `implicit class XtensionTermAssignSyntax` as a naming convention for implicit classes.